src/users/users.prisma

```diff
@@ -1,23 +1,19 @@
 //
-//  DO NOT MODIFY THIS FILE!
+//  This file is generated by Prisma from the TypeORM schema.
```

Comment:
This is obvious enough that it doesn't need to be stated in the document

Reply:
accepted in commit https://github.com/SageSeekerSociety/cheese-backend/pull/226/commits/d87a714211f1dd9be5f758161c695eb681dea5d4